Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Memory.yaml

```diff
@@ -1371,7 +1371,7 @@
       help         : >
                      In-Band ECC Operation Mode
       length       : 0x01
-      value        : 0x00
+      value        : 0x02

```

Comment:
Is the intention here to enable IBECC for all SKUs/boards by default ?

Reply:
Yes to enable it for all sku by default

Reply:
Yes Maurice, fusa team requested this